src/log-store/src/kafka/client_manager.rs

```diff
@@ -12,17 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
 use std::sync::Arc;
 
 use common_config::wal::{KafkaConfig, KafkaWalTopic as Topic};
```

Comment:
This alias is confusing... I realize that it isn't rskafka's Topic...
https://docs.rs/rskafka/latest/rskafka/topic/struct.Topic.html

Reply:
`Topic` will be renamed in an independent PR.